src/core/ruby_version.rs

```diff
@@ -416,7 +425,29 @@ macro_rules! get_stack_trace(
             let mut trace = Vec::new();
             let cfps = get_cfps(thread.cfp as usize, stack_base(&thread) as usize, source)?;
             for cfp in cfps.iter() {
-                if cfp.iseq as usize == 0 {
+                let cfunc = ruby_frame_cfunc(&cfp, source)?;
```

Comment:
I flipped the conditional around here and got rid of the frame flag filtering to match what ruby's own backtrace function is doing.